bin/additional-ci-steps

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Exit immediately if any command returns a non-zero status.
+set -e
+
+# Install node
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get install -y nodejs
+
+# Install yarn
+curl -o- -L https://yarnpkg.com/install.sh | bash
+source ~/.bashrc
+export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"
+
+# Install npm packages
+yarn install
+
```

Comment:
Should we also run `yarn lint` in CI?

Reply:
We've been doing pre-commit hooks, pre-push hooks, and running lint in CI in different repos. If this is OSS, I think CI is a great place for a lint check.